functests/4_latency/latency.go

```diff
@@ -255,6 +228,65 @@ var _ = Describe("[performance] Latency Test", func() {
 	})
 })
 
+//read LATENCY_TEST_RUN environment var and parse it if set
+func getLatencyTestRun() {
+	latencyTestRunEnv := os.Getenv("LATENCY_TEST_RUN")
+	if latencyTestRunEnv != "" {
+		if latencyTestRunEnv == "true" {
+			latencyTestRun = true
+		}
+	}
+}
+
+//read LATENCY_TEST_RUNTIME environment var and parse it if set
+func getLatencyTestRuntime() {
+	latencyTestRuntimeEnv := os.Getenv("LATENCY_TEST_RUNTIME")
+	if latencyTestRuntimeEnv != "" {
+		latencyTestRuntime = latencyTestRuntimeEnv
+	}
+}
+
+//read LATENCY_TEST_DELAY environment var and parse it if set
+func getLatencyTestDelay() {
+	latencyTestDelayEnv := os.Getenv("LATENCY_TEST_DELAY")
+	if latencyTestDelayEnv != "" {
+		var err error
+		if latencyTestDelay, err = strconv.Atoi(latencyTestDelayEnv); err != nil {
+			klog.Fatalf("the environment variable LATENCY_TEST_DELAY has incorrect value %q", latencyTestDelayEnv)
+		}
+	}
+}
+
+//read LATENCY_TEST_CPUS environment var and parse it if set
+func getLatencyTestCpus() {
+	latencyTestCpusEnv := os.Getenv("LATENCY_TEST_CPUS")
+	if latencyTestCpusEnv != "" {
+		var err error
+		if latencyTestCpus, err = strconv.Atoi(latencyTestCpusEnv); err != nil {
+			klog.Fatalf("the environment variable LATENCY_TEST_CPUS has incorrect value %q", latencyTestCpusEnv)
+		}
+	}
+}
+
+//read MAXIMUM_LATENCY environment var and parse it if set
+func getMaxLatency() {
+	unifiedMaxLatencyEnv := os.Getenv("MAXIMUM_LATENCY")
+	if unifiedMaxLatencyEnv != "" {
+		var err error
+		if maximumLatency, err = strconv.Atoi(unifiedMaxLatencyEnv); err != nil {
+			klog.Fatalf("the environment variable MAXIMUM_LATENCY has incorrect value %q", unifiedMaxLatencyEnv)
+		}
+	}
```

Comment:
Same here, instead of saving the parsed value, return it.

Reply:
fixed, thanks.